### ISSUE TITLE
Cannot delete items within workflow overview panel

### ISSUE THREAD

**Issue by charliestrafe** [engram-design/Workflow#154]
### Describe the bug

I have an issue where the workflow overview panel is not letting me delete items that were previously submitted for review.

I can select the item in the table, but then when I click the dropdown to delete it, the 'delete' item is always grayed out and unavailable to click. See screenshot below
![Screenshot 2022-07-13 at 08 56 33](https://user-images.githubusercontent.com/47243887/178684054-902e9a7e-d4e3-4e4b-a266-b26812046df9.png)

### Steps to reproduce

1. Submit an item for review
2. Access the workflow overview panel
3. Attempt to select and delete the item

### Craft CMS version

Craft Pro 4.1.2

### Plugin version

dev-craft-4 as 2.0.0-beta.4

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by charliestrafe** [comment]
This is now working

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.5](https://github.com/verbb/workflow/releases/tag/2.0.0-beta.5)
